dashboard/test/models/peer_review_test.rb

```diff
@@ -49,14 +46,6 @@ class PeerReviewTest < ActiveSupport::TestCase
     assert_equal Set[nil, 'escalated'], PeerReview.where(submitter: @user, level: @level).map(&:status).to_set
   end
 
-  test 'submitting a peer reviewed level when I am not enrolled in the module should not create PeerReview objects' do
-    Plc::EnrollmentModuleAssignment.stubs(:exists?).returns(false)
-
-    assert_no_difference('PeerReview.count') do
-      track_progress @level_source.id
-    end
-  end
```

Comment:
We're simplifying the business logic, so this test case goes away entirely (as does some associated setup and detail in other test cases).